### ISSUE TITLE
Video 3 in BC of week 5

### ISSUE THREAD

**Issue by stefaniemuff** [opetchey/BIO144#49]
I just noticed that in the BC video 3 of week 5 there is a problem with the interpretation of the significance of the interaction term (earthworm example, around minute 13). Actually one would have to use the anova table again to test the interaction between Magenumf and Gattung and not the single p-values.

(Btw, for next  year I'll plan to bring an example of an interaction term with at least 3 levels already in the lecture, because I have now only used a binary variable with interaction in lecture 4...)

**Comment by opetchey** [comment]
I agree that this is better in general. In practice here it makes no difference, so I view this as an enhancement rather than essential change.

The enhancement would be to redo the video with ANOVA table and not summary.
Should be done alongside https://github.com/opetchey/BIO144/issues/53
